Author: Andreas Bombe <aeb@debian.org>
Description: Disable check for new upstream version
 The Debian packaged anki should not keep notifying the user about new upstream
 versions.  For the user, Debian is upstream and new versions of the package
 would be found through the usual channels (e.g. aptitude).
Forwarded: not-needed

Index: git_anki/aqt/main.py
===================================================================
--- git_anki.orig/aqt/main.py	2013-11-04 22:31:18.500009310 +0100
+++ git_anki/aqt/main.py	2013-11-04 22:31:18.498009306 +0100
@@ -811,6 +811,7 @@
     ##########################################################################
 
     def setupAutoUpdate(self):
+        return  # do not lookup latest upstream version in Debian packaged anki
         import aqt.update
         self.autoUpdate = aqt.update.LatestVersionFinder(self)
         self.connect(self.autoUpdate, SIGNAL("newVerAvail"), self.newVerAvail)
